### ISSUE TITLE
GitHub vs. SourceForge as preferred platform

### ISSUE THREAD

**Issue by Socob** [texstudio-org/texstudio#984]
This is related (but not identical) to #93. It seems that the GitHub repository https://github.com/texstudio-org/texstudio and the SourceForge project https://sourceforge.net/projects/texstudio/ are still being run in parallel. However, neither page indicates which of these is to be preferred and, especially, where bugs/issues should be reported. The most recent activity in the [GitHub issues](https://github.com/texstudio-org/texstudio/issues) is 9 hours ago, while the most recent report on the [SourceForge bug tracker](https://sourceforge.net/p/texstudio/bugs/) is 8 hours ago. I think it would help alleviate a lot of confusion if a statement could be added to both project pages indicating where new reports should be directed.

**Comment by timhoffm** [comment]
Current development is going on at github. This is noted https://texstudio.org/.

Unfortunately, we did not have any notice on that on sourceforge. AFAIR partly we forgot, partly because sf had a time where they had [shady practices of handling abandoned projects](https://www.ghacks.net/2015/06/01/sourceforge-adds-and-removes-adware-installers-to-abandoned-projects/).

I've now edited the sf site:

![grafik](https://user-images.githubusercontent.com/2836374/77011052-4ea3db80-696b-11ea-80be-91f7f03daf50.png)
![grafik](https://user-images.githubusercontent.com/2836374/77011077-5c596100-696b-11ea-9a79-d925bb2fa2d3.png)

I don't think we can put messages at the top of the ticket pages.
